### ISSUE TITLE
'Back' arrow when uploading files is a long process to get back to program selection

### ISSUE THREAD

**Issue by TiCampbell** [Get-Your/GetFoco#67]
The 'back' arrow in the 'file upload' step uses `history.back()`, which makes it potentially a long process to get back to the program selection. Either need to simplify the page loads between 'program selection' and 'file upload' or change the 'back' arrow to link directly to the 'program selection' page.

**Comment by TiCampbell** [comment]
@penoptech I can't find this having changed in `dev`...?

**Comment by TiCampbell** [comment]
Ah, unless this in 'in review' because it's complete in it's feature branch. If that's the case, I'll create a new Status to reflect that and conform better to our new way of organizing releases. Let me know.

**Comment by penoptech** [comment]
Good catch! I incorrectly resolved a merge conflict that caused a couple different issues that you noted in some other issues. The fix is in the `TEST` environment after you helped me redeploy.
